docker-compose.yml

```diff
@@ -170,26 +158,29 @@ services:
       STORE_PGDATABASE: dev_database
 
       ADMIN_API_PORT: 3001
-      STORE_API: http://store-api:3000
+      STORE_API: http://stpre-api:3000
```

Comment:
There is a typo here in the url. `stpre` => `store`

Reply:
Fixed.